metrics/statusMetrics.go

```diff
@@ -0,0 +1,59 @@
+package metrics
+
+import (
+	"strings"
+	"sync"
+)
+
+//const (
+//	numWSConnections  = "num_ws_connections"
+//	numIndexingErrors = "num_indexing_errors"
+//)
+
+type statusMetrics struct {
```

Comment:
I assume that in an upcoming PR, this component will be injected into internal processors so it can be fed with the data

Reply:
Exactly